backend/index.js

```diff
@@ -0,0 +1,97 @@
+const express = require('express');
+const mysql = require('mysql2');
+const app = express();
+
+app.use(express.json());
+
+// Configuration de la connexion MySQL
+const db = mysql.createConnection({
+    host: 'database', // Le nom du service MySQL défini dans docker-compose
+    user: 'root',
+    password: 'root',
+    database: 'backrow'
+});
+
+// Connexion à la base de données
+db.connect((err) => {
+    if (err) {
+        console.error('Erreur de connexion à MySQL :', err);
+        return;
+    }
+    console.log('Connecté à la base de données MySQL');
```

Comment:
comme t'utilise pas d'orm verifie que ta table soit bien crée ici avec un CREATE TABLE IF NOT EXISTS